guides/cookbook.md

```diff
@@ -31,141 +80,177 @@ constructing input streams to read files), which is out of scope for this
 cookbook. Code such as this will be replaced by a method with an empty (but
 implied) implementation.
 
-> *Java*
-> 
-> Import statements for classes internal to the Ion library or to `java.lang`
-> are omitted. Other external classes will either be first referenced by their
-> fully-qualified names, or will be preceded by an import statement. Import
-> statements have global scope.
+## Reading and Writing Ion Data
+
+<script>writeTabs()</script>
+<div class="tabpane Java" markdown="1">
+Implementations of the [`IonReader`][4] and [`IonWriter`][5] interfaces are
```

Comment:
Huh, this is nice! I didn't realize MarkDown supported this [footnote-as-href](https://daringfireball.net/projects/markdown/syntax#link) notation. It keeps the source much more readable than having all of the URLs inline.